pkgs/applications/misc/furtherance/default.nix

```diff
@@ -6,19 +6,19 @@
 
 stdenv.mkDerivation rec {
```

Comment:
A nice to have would be to use the `finalAttrs` pattern here.

Let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

It would be nice to do it within this PR (in a new commit), but if you don't want to do it, it's ok ;)

Reply:
i'll give it a shot!

Reply:
Excellent, thanks !

Reply:
Donion rings, thanks for the advice and help!